lib/puppet/provider/package/pacman.rb

```diff
@@ -25,6 +25,7 @@ def self.yaourt?
   has_feature :uninstall_options
   has_feature :upgradeable
   has_feature :virtual_packages
+  has_feature :purgeable
```

Comment:
I think this isn't even required and somehow automatically enabled when a `purge` method exists? but some providers explicitly enable it?

Reply:
For posterity, since the `package` type defines the `purgeable` feature as depending on the `purge` method https://github.com/puppetlabs/puppet/blob/12cc9e66ee8609b4742a84342fe305f70430bed7/lib/puppet/type/package.rb#L47-L51

Then it is not strictly necessary to specify `has_feature` due to the `available` check in https://github.com/puppetlabs/puppet/blob/12cc9e66ee8609b4742a84342fe305f70430bed7/lib/puppet/util/provider_features.rb#L156

See related commits 5b2ffbcb5d2e4113efec9c8ba882c5506d53c242 and 3d17685f9954b584cf84a6fe224b2513007108f0